sds.h

```diff
@@ -34,7 +34,7 @@
 #define __SDS_H
```

Comment:
I think we pull sds directly from redis upstream.. so this might get overwritten at a later point in time.

Reply:
AFAICT the source for sds has already diverged from the upstream version (due to previous hiredis patches). This patch merely fixes regressions due to earlier patches.

Going forward it would be good to sync with/upstream to redis.